internal/sgx/sgx.go

```diff
@@ -249,6 +269,11 @@ func (ctx *SgxContext) RemoveSigner(name string) error {
 
 	s := ctx.signers.Get(name)
 	if s == nil {
+		if quoteInfo, ok := ctx.quotes[name]; ok {
+			ctx.p11Ctx.DestroyObject(ctx.p11Session, quoteInfo.prvKeyHandle)
```

Comment:
Is there a need to check if the handles exist?

Reply:
I donät think `quoteInfo` with invalid handles gets stored in the map.